streams/src/test/java/org/apache/kafka/streams/utils/TestUtils.java

```diff
@@ -90,7 +90,7 @@ public static String safeUniqueTestName(final TestInfo testInfo) {
         return safeUniqueTestName(methodName);
     }
 
-    private static String safeUniqueTestName(final String testName) {
+    public static String safeUniqueTestName(final String testName) {
```

Comment:
Needed this since I need to generate unique test names without the `@BeforEach` annotation which passes in the `TestInfo` object.  Is there any reason this couldn't be public anyway?